### PR TITLE
Create 2019-02-11--go-core-dev-team-weekly.md

### PR DESCRIPTION
**Pull request by eingenito**

### CODE REVIEW

File: meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 2019-02-11
+
+Please note. There has been a  [meeting format change](https://github.com/ipfs/team-mgmt/issues/827#issuecomment-452157617) intended to make our brief sync time more valuable and more easily understood by new/occasional attendees.
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md]: Nice 👏🏽

File: meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  - Are you blocked waiting for input?
+  - PRs for review? Remember the [project board](https://github.com/orgs/ipfs/projects/1)
+- Review progress on major initiatives
+  - Find or create the section for the IPFS initiative(s) you worked on.
+  - Summarize the initiative briefly in notes to give a little context.
+  - Summarize your progress
+  - Summarize your next steps.
+- Review other Questions/Announcements/Notes
+
+## Current IPFS Initiatives
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md]: Perhaps rename to "Current Go Core Initiatives" as these are representative of what this team is focusing on

File: meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+#### Datastore
+Summary: We are investigating various IPFS datastore options with the goal of replacing the current default datastore with a faster, more scalable one: [Epic Issue](https://github.com/ipfs/go-ipfs/issues/4279)
+
+@eingenito - This issue is blocked (ish?) on benchmarking infrastructure. There may be several other outstanding issues that need to be addressed before we even have a go/no-go on Badger. Epic may need a refresh to clean out closed issues and add new ones. 
+
+@magik6k - Moved go-ds-bench a bit forward, can start running serious benchmarks soon
+  - Needs infra: https://github.com/protocol/ad-hoc-infra/pull/24
+    - This is dev-only, will need more & more diverse infra soon
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md]: @eingenito @magik6k Is there a requirements doc for what this benchmarking infrastructure is? Would providing 4TB to each member of the team enable you to start pulling some results?

File: meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+#### Bitswap
+Summary: We are improving the performance of transferring data between IPFS instances: [Epic Issue](https://github.com/ipfs/go-ipfs/issues/5723)
+
+- all refactor tickets are merged. cant remember if that was last week.
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md]: Is there benchmarks for large datasets between two nodes? 

The perf improvements should be visible on the interop tests http://github.com/ipfs/interop

File: meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+- More work on peermanager + optimization of sessions... but
+- @hannahhoward mostly on graphsync right now
+
+#### CoreAPI
+Summary: We are improving the API used to drive IPFS functionality and extracting it to a separate project without a dependency on go-ipfs itself. [Epic Issue](https://github.com/ipfs/go-ipfs/issues/4498)
+
+- Interface has been extracted from go-ipfs! https://github.com/ipfs/interface-go-ipfs-core/
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md]: 👏🏽👏🏽👏🏽

Comment (daviddias) [reply meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md]: Is there a tutorial on how to use go-ipfs through importing it as a package? Analogous to: https://github.com/ipfs/js-ipfs/tree/master/examples#tutorials

File: meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+
+## Questions
+
+* Jenkins/Travis? (@eingenito)
+Turns out the go-ipfs project is okay to use CircleCI exclusively
+* Graphsync (@eingenito)
+Graphsync team is busy this week but will try to arrange some time to further discuss design
+* Base32/CIDv1 (@eingenito)
+Most important next milestone is storing/fetching multihashes in the DHT in libp2p
+
```
Comment (daviddias) [inline meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md]: This notes are amazing! It is amazing to see progress so clearly outlined. Awesome work Go Core Team 👏🏽👏🏽👏🏽👏🏽

File: meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md (hunk 3 above)
Comment (magik6k) [reply meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md]: This is a small scale infrastructure where I want to test a diverse set of configurations, to tinker with tooling and different performance characteristics, mostly to know where 'more testing is needed'.

Testing on large data can produce interesting outcomes, but tends to be predictable from small scale data. I want to start by testing a lot of scenarios to see what will tell us the most when scaled up

File: meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md (hunk 5 above)
Comment (magik6k) [reply meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md]: We still need to cleanup/refactor go-ipfs constructor, which iirc is OKR this quarter

**Pull request merged**